src/migrations/20221124123914-add-favorites.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(
+        `
+            CREATE TABLE IF NOT EXISTS favorite_features
+            (
+                feature varchar(255) not null references features (name) on DELETE CASCADE,
```

Comment:
I'd use upper case letters for SQL stuff and lower case for our stuff. It's easier for the reader to parse it like this.
e.g. feature VARCHAR(255) NOT NULL REFERENCES features (name) ON DELETE CASCADE

Reply:
Now that this is merged, it is not best to edit the migration. Will keep it in mind for future